packages/data-explorer/tests/unit/fixtures/state.ts

```diff
@@ -0,0 +1,71 @@
+import ApplicationState from '@/types/ApplicationState'
```

Comment:
why use a huge actual context object , we do not need to test context-ui , only the hasRole